src/main/scala/mesosphere/marathon/Exception.scala

```diff
@@ -31,6 +31,12 @@ case class AppLockedException(deploymentIds: Seq[String] = Nil)
       "View details at '/v2/deployments/<DEPLOYMENT_ID>'."
   )
 
+case class TooManyRunningDeploymentsException(maxNum: Int) extends Exception(
+  s"Max number ($maxNum) of running deployments is achieved. Wait for existing deployments to complete or cancel one." +
+    "You can increase max deployment number by using --max_running_deployments parameter but be " +
+    "advised that this can have negative effects on the performance."
```

Comment:
`on performance`?

Reply:
Grammarly disagrees with you ;)

Reply:
Can I assume that you used Google Translate to write it then? :)
Both variants are correct, since both of them depend on the meaning, and according to how I understand it, I wouldn't use `the`.